selene-lib/src/lib.rs

```diff
@@ -296,6 +296,7 @@ use_rules! {
     ifs_same_cond: rules::ifs_same_cond::IfsSameCondLint,
     incorrect_standard_library_use: rules::standard_library::StandardLibraryLint,
     invalid_lint_filter: rules::invalid_lint_filter::InvalidLintFilterLint,
+    limit_function_complexity: rules::limit_function_complexity::LimitFunctionComplexityLint,

```

Comment:
Nit: lints are named after the problem. For instance, "allow mismatched arg count". Should rename to something like `overly_complex_function` or similar.

Reply:
Stole the rule name from luacheck, and tweaked the message with align a bit better with luacheck, which complies with your feedback above. let me know if you want to iterate on it more.